Add chip clip

brentyi opened the pull request:
Hi Alex!

Are you taking contributions to your examples directory? I made a bag clip and feel it needs a home.

Can also add some photos later.

![image](https://user-images.githubusercontent.com/6992947/133168534-f2ef3776-819d-4ecd-9bd7-879e00dcd21e.png)

unixpickle commented:
This looks awesome! I'm super glad someone else has gotten some use out of this library. I'd love to add this to the examples :)

A few suggestions before merging
 * In Go, it is customary to run `gofmt` on code. I'd suggest setting this up in your editor, but for this single file it suffices to just run `gofmt -w file.go`. I'll add a test for this so it gets checked automatically in future PRs.
 * I wouldn't commit the STL file itself, as it is large so it's not practical to provide them for every example.
 * I typically run [smallpng](https://github.com/unixpickle/smallpng) on every rendering image to keep down file sizes in the repo. Your render already looks pretty small, but might be worth trying.

brentyi commented:
Done!

Re-formatted, removed the STL, and ran the color reduction on the PNG.

Added some photos as well:
![image](https://user-images.githubusercontent.com/6992947/133342716-975244b3-b7fc-44f6-85bd-1aeb11860dfd.png)
